client/electron/src/setup.ts

```diff
@@ -65,13 +68,24 @@ export class ElectronCapacitorApp {
     this.TrayMenuTemplate = [
       new MenuItem({
         label: 'Quit App',
-        click: () => {
+        click: async () => {
           this.forceClose = true;
-          app.quit();
+          await this.quitApp();
         },
       }),
     ];
 
+    log.initialize();
+    log.info('Starting app');
+    Object.assign(console, log.functions);
+
+    if (process.platform === 'win32') {
+      const reg = require('regedit');
+      const vbsDirectory = join(app.getAppPath(), '../vbs');
```

Comment:
```suggestion
      const vbsDirectory = join(app.getAppPath(), '../node_modules/regedit/vbs');
```

I think that should work :thinking: